plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -145,7 +147,8 @@ class PodDebugTabView extends React.Component {
     let mainContent = null;
     let offerCount = null;
 
-    if (queue == null || queue.declinedOffers.summary == null) {
+    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(queue)
+      || queue.declinedOffers.summary == null) {
```

Comment:
Yeah, we can definitely clean these up! I created a JIRA: https://mesosphere.atlassian.net/browse/DCOS-13205